.github/workflows/release.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:        
+      - v*
+
+jobs:
```

Comment:
what I was suggesting earlier was to still call `lint` & `test` in the release flow but somehow not dedup the code/action (which was a secondary concern). we will still need to test the code/image before releasing it (may sound redundant since the pushed code should have passed CI, but sometimes ... )

Reply:
Yes, this is what I thought before, we need to have some basic checks in case something weird happened.

Reply:
can you please add that lint and test back